react/features/toolbox/defaultToolbarButtons.web.js

```diff
@@ -228,15 +241,26 @@ export default function getDefaultButtons() {
             enabled: true,
             id: 'toolbar_button_fullScreen',
             onClick() {
-                sendAnalyticsEvent(TOOLBAR_FULLSCREEN_ENABLED);
+                // TODO: why is the fullscreen button handled differently than
+                // the fullscreen keyboard shortcut (one results in a direct
+                // call to toggleFullScreen, while the other fires an
+                // UIEvents.TOGGLE_FULLSCREEN event)?
+
+                // TODO: include an 'enable' attribute which specifies whether
+                // pressing the shortcut resulted in fullscreen being enabled
+                // or disabled.
+                sendAnalytics(createToolbarEvent('fullscreen'));
 
                 APP.UI.emitEvent(UIEvents.TOGGLE_FULLSCREEN);
             },
             shortcut: 'S',
             shortcutAttr: 'toggleFullscreenPopover',
             shortcutDescription: 'keyboardShortcuts.fullScreen',
             shortcutFunc() {
-                sendAnalyticsEvent('shortcut.fullscreen.toggled');
+                // TODO: include an 'enable' attribute which specifies whether
+                // pressing the shortcut resulted in fullscreen being enabled
+                // or disabled.
```

Comment:
Why not complete right now these TODOs to add more analytics?

Reply:
I completed most of them